jib-core/src/test/java/com/google/cloud/tools/jib/builder/steps/PushImageStepTest.java

```diff
@@ -89,6 +97,68 @@ public void testMakeListForManifestList() throws IOException, RegistryException
     }
   }
 
+  @Test
+  public void testMakeList_multiPlatform_enabled() throws IOException, RegistryException {
+    Image asd = Image.builder(V22ManifestTemplate.class).setArchitecture("wasm").build();
+
+    Mockito.when(containerConfig.isPlatformTag()).thenReturn(true);
+
+    ImmutableList<PushImageStep> pushImageStepList =
+        PushImageStep.makeList(
+            buildContext,
+            progressDispatcherFactory,
+            registryClient,
+            new BlobDescriptor(mockDescriptorDigest),
+            asd,
+            false);
+
+    ArgumentCaptor<String> tagCAtcher = ArgumentCaptor.forClass(String.class);
+    Mockito.when(registryClient.pushManifest(Mockito.any(), tagCAtcher.capture())).thenReturn(null);
+
+    Assert.assertEquals(2, pushImageStepList.size());
+    for (PushImageStep pushImageStep : pushImageStepList) {
+      BuildResult buildResult = pushImageStep.call();
+      Assert.assertEquals(
+          "sha256:0dd75658cf52608fbd72eb95ff5fc5946966258c3676b35d336bfcc7ac5006f1",
+          buildResult.getImageDigest().toString());
+      Assert.assertEquals("mockDescriptorDigest", buildResult.getImageId().toString());
+    }
+    Set<String> allValues = ImmutableSet.copyOf(tagCAtcher.getAllValues());
+    Set<String> expectedTags = ImmutableSet.of("tag1-wasm", "tag2-wasm");
+    Assert.assertEquals(expectedTags, allValues);
```

Comment:
```suggestion
    assertThat(tagCatcher.getAllValues()).containsExactly("tag1-wasm", "tag2-wasm")
```
We also need to add `import static com.google.common.truth.Truth.assertThat;` to the import statements.